pkgs/tools/text/xml/jing-trang/default.nix

```diff
@@ -1,17 +1,20 @@
-{ lib, stdenv, fetchFromGitHub, jre_headless, jdk, ant, saxon }:
+{ lib, stdenv, fetchFromGitHub, jre_headless, jdk, ant, saxon, git }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "jing-trang";
-  version = "20151127";
+  version = "20181222";
 
   src = fetchFromGitHub {
     owner = "relaxng";
     repo = "jing-trang";
-    rev = "47a0cbdaec2d48824b78a1c19879ac7875509598"; # needed to compile with jdk8
-    sha256 = "1hhn52z9mv1x9nyvyqnmzg5yrs2lzm9xac7i15izppv02wp32qha";
+    rev = "V${version}";
+    sha256 = "sha256-cQ1yd8k8qv8OQxDus1nUU2LUQBswQRXX/Rif88Xl5gY=";
+    leaveDotGit = true;
```

Comment:
Is leaveDotGit be reproducible? (I think not.)

Reply:
I patched the build process so it does not rely on `git`.  `leaveDotGit = true;` is no more needed.